### PR TITLE
update outer template files

### LINKED ISSUE #7: Update AsciidoctorPDF and PDFTheme
Update asciidoctor PDF, the PDF Theme submodule and gradle.

### LINKED ISSUE #8: Adjust file suffix
Adjust file suffix (- instead of _) (see build.gradle in advanced template repository)

### PR DESCRIPTION
**Pull request by larsrh**
Resolves #7

Resolves #8

### CODE REVIEW

File: .vscode/tasks.json [hunk 1]
```diff
@@ -0,0 +1,16 @@
+{
```
Comment (programming-wolf) [inline .vscode/tasks.json:1]: I guess this file was added by accident? `.vscode` should be added to .gitignore

**Review by programming-wolf — CHANGES_REQUESTED** [review]

File: .vscode/tasks.json (hunk 1 above)
Comment (larsrh) [reply .vscode/tasks.json:1]: Seems on purpose to me: https://github.com/isaqb-org/advanced-template/commit/bed5bef4622e57ad455b1acc7786893ba05f8159

Comment (programming-wolf) [reply .vscode/tasks.json:1]: Apparently, yes....

**Review by programming-wolf — APPROVED** [review]
Ok.

**Pull request merged**